en/docs/learn/examples/scheduled-tasks/task-scheduling-simple-trigger.md

```diff
@@ -22,35 +19,43 @@ Following are the integration artifacts that we can used to implement this scena
                 </m0:request>
             </m0:getQuote>
         </property>
+        <property xmlns:task="http://www.wso2.org/products/wso2commons/tasks" name="injectTo" value="sequence"/>
+        <property xmlns:task="http://www.wso2.org/products/wso2commons/tasks" name="sequenceName"   value="main"/>
```

Comment:
Let's rename the sequence name. The 'main' is a special sequence in older versions. Eventhough it is not there now, it is better to avoid using it.